### DISCUSSION TITLE
github.com app webhook to self-hosted renovate on private aws eks cluster

### DISCUSSION THREAD

**Discussion by dhorner71** [renovatebot/renovate#33355] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

github renovate ce v9

### Please tell us more about your question or problem

we are in the process of migrating to github.com from github onprem enterprise server.  i've migrated our repo and have configured github.com app successfully except for the github app webhook back to self hosted renovate thru aws.  i know the webhook is not necessary.

a few questions:
-can renovate be configured to poll for dependency dashboard or PR events rather than implementing a webhook?
-has anyone implemented a "webhook" replacement back thru aws with elevated auths to private resources instead of the standard webhook secret string to public resource? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
1. No
